.ci/obfuscate/gnirts.js

```diff
@@ -0,0 +1,29 @@
+'use strict'
+/*********************************************************************
+ * Copyright (c) 2020 Red Hat, Inc.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ **********************************************************************/
+
+const gnirts = require('gnirts');
+const fs = require('fs')
+
+const rootDir = process.cwd()
+
+const obfuscateJSFiles = () => {
+  const dirFilesToObfuscate = [ 'lib/hooks/analytics/analytics.js' ]
+ 
+  dirFilesToObfuscate.forEach((jsFilePath)=> {
+    if (fs.existsSync(`${rootDir}/${jsFilePath}`)) {
```

Comment:
${rootDir}/${jsFilePath} looks like platform specific delimiter `/`. Maybe you can use [path.join](https://nodejs.org/api/path.html#path_path_join_paths)?